x-pack/plugins/security_solution/server/endpoint/services/fleet/endpoint_fleet_services_factory.ts

```diff
@@ -48,7 +48,7 @@ export class EndpointFleetServicesFactory implements EndpointFleetServicesFactor
       packages: packageService.asScoped(req),
       packagePolicy,
 
-      asInternal: this.asInternalUser.bind(this),
+      asScoped: this.asScoped.bind(this),
```

Comment:
please revert. This was by design to provide the ability for the caller to still access an `internal` client if needed.

Reply:
I see. Will revert so that `asScoped` has the `internal` client.

Reply:
reverted 4f0ac7a73c260fe5b9c33ee65249ec9ba754d7aa